proto/storage/report_configuration.proto

```diff
@@ -34,6 +34,7 @@ message ReportConfiguration {
     ResourceScope                   resource_scope              = 11;
     repeated NotifierConfiguration  notifiers                   = 12;
     SlimUser                        creator                     = 13 [(gogoproto.moretags) = 'sql:"ignore_labels(User ID)"'];
+    int32                           version                     = 14;
```

Comment:
Add a comment that this was added in 4.3 and a value of 0 would mean that the config is un-migrated and from a version before 4.2